index.bs

```diff
@@ -703,34 +751,74 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator with which to complete the operation.
 </div>
 
+### <dfn interface>AuthenticatorResponse</dfn> interfaces ### {#iface-authenticatorresponse}
 
-## Information about Scoped Credential (interface <dfn interface>ScopedCredentialInfo</dfn>) ## {#iface-credentialInfo}
+The {{AuthenticatorResponse}} interface and its derieved interfaces represent the data 
```

Comment:
Thanks! Fixed in https://github.com/w3c/webauthn/pull/384/commits/fedccedb468282665936264105fac6ee09e577cf.